src/frontend/js/modules/header/header.js

```diff
@@ -118,6 +118,18 @@
           scope.domainId = session.domain._id;
           scope.baseUrl = window.openpaas && window.openpaas.OPENPAAS_API_URL || '';
```

Comment:
```suggestion
```

this is no longer needed I guess.

Reply:
@oubchid please remove this line ( it was used in the template ng-src attribute )

Reply:
@oubchid any update ?